corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -447,8 +447,11 @@ hqDefine("cloudcare/js/formplayer/menus/views", [
                 arrow.removeClass("fa-angle-double-up");
                 arrow.addClass("fa-angle-double-down");
                 tileContent.addClass("collapsed-tile-content");
+                const scrollContainer = $(constants.SCROLLABLE_CONTENT_CONTAINER);
```

Comment:
nit: this could be `const scrollContainer = this.scrollContainer();` for consistency